app/models/procedure.rb

```diff
@@ -32,13 +32,10 @@ class Procedure < ActiveRecord::Base
   mount_uploader :logo, ProcedureLogoUploader
 
   default_scope { where(hidden_at: nil) }
-  scope :brouillons, -> { not_published.not_archived }
+  scope :brouillons, -> { where(published_at: nil).where(archived_at: nil) }
```

Comment:
je suppose qu'il n'est pas possible de faire where.not.published.where.not.archivee ?